internal/service/reclaimspace/reclaimspace.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+package service;
+
+option go_package = "github.com/csi-addons/kubernetes-csi-addons/internal/service";
+
+// ReclaimSpace holds the RPC menthod for allowing the
+// communication between the CSIAddons controller and the sidecar.
+service ReclaimSpace {
+    // ControllerReclaimSpace is a procedure that gets called on the CSI
+    // sidecar.
+    rpc ControllerReclaimSpace (ControllerReclaimSpaceRequest) returns(ReclaimSpaceResponse) {}
+    // NodeReclaimSpace is a procedure that gets called on the CSI
+    // sidecar.
+    rpc NodeReclaimSpace (NodeReclaimSpaceRequest) returns(ReclaimSpaceResponse) {}
+
+  }
+// ControllerReclaimSpaceRequest contains the information i.e., pv_name
+// received from the CSIAddons controller for performing the reclaim operation.
+message ControllerReclaimSpaceRequest {
+    // The name of the pv. This field is REQUIRED.
+    string pv_name = 1;
+}
+
+// SidecarCSIAddonsResponse holda the information about the result of the
```

Comment:
Please review the comments, `SidecarCSIAddonsResponse` is not correct (and `holda` typo)